SPECS-EXTENDED/python-enchant/python-enchant.spec

```diff
@@ -0,0 +1,311 @@
+%global srcname enchant
+
+Name:           python-enchant
+Version:        3.2.2
+Release:        7%{?dist}
+Summary:        Python bindings for Enchant spellchecking library
```

Comment:
Maybe we should put summary as the first, as per our convention :)

Reply:
Fixed